src/compute-client/src/controller.rs

```diff
@@ -926,11 +916,7 @@ pub struct CollectionState<T> {
     pub compute_dependencies: Vec<GlobalId>,
 
     /// Reported progress in the write capabilities.
-    ///
-    /// Importantly, this is not a write capability, but what we have heard about the
-    /// write capabilities of others. All future writes will have times greater than or
-    /// equal to `write_frontier.frontier()`.
-    pub write_frontier: MutableAntichain<T>,
+    pub write_frontier: Antichain<T>,
```

Comment:
Can you point out a scenario where multiple write capabilities would be a useful thing to have?

Reply:
Hypothetically, we might want to allow all collections (even views) to be modifiable by e.g. a controller user (say Adapater). A write capability would allow that, in addition to the frontier updates coming back from `computed`. Not sure that this makes any sense at the moment, but it is one of the reasons it exists! But also, we can remove it and add it back later. More worried about storage controller version, and some amount of maintaining parity between the two.